src/bot.ts

```diff
@@ -33,6 +36,9 @@ export default class {
     this.shouldRegisterCommandsOnBot = config.REGISTER_COMMANDS_ON_BOT;
     this.commandsByName = new Collection();
     this.commandsByButtonId = new Collection();
+    this.cbot_status = config.BOT_STATUS;
```

Comment:
Let's add a new `.config` property to this class so we don't have to extract all of these out.

Reply:
bot.ts
```
constructor(
  @inject(TYPES.Client) client: Client,
    @inject(TYPES.Config) config: Config,
      @inject(TYPES.ConfigStatus) configStatus: Config,
  ) {
    this.client = client;
    this.token = config.DISCORD_TOKEN;
    this.shouldRegisterCommandsOnBot = config.REGISTER_COMMANDS_ON_BOT;
    this.commandsByName = new Collection();
    this.commandsByButtonId = new Collection();
    this.cbot_status = configStatus.BOT_STATUS;
    this.cbot_act_type = configStatus.BOT_ACTIVITY_TYPE;
    this.cbot_act = configStatus.BOT_ACTIVITY;
  }
```

inversify.config.ts
```
// Config values
container.bind(TYPES.Config).toConstantValue(new ConfigProvider());
container.bind(TYPES.ConfigStatus).toConstantValue(new ConfigProvider());

```

types.ts
```ConfigStatus: Symbol('ConfigStatus'),````

Like this?

Reply:
Changed it in the newest [commit](https://github.com/codetheweb/muse/pull/599/commits/e3b1d61ddd1328db29ee1b1ef7d2fbaa49dfeb99). Seems to work fine and looks better.